test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -36,76 +36,22 @@ extern std::string const c_testExampleBlockchainTest;
 extern std::string const c_testExampleRLPTest;
 
 //Main Test functinos
-void fillRandomTest(std::function<void(json_spirit::mValue&, bool)> _doTests, std::string const& _testString, bool _debug = false);
+void fillRandomTest(std::function<void(json_spirit::mValue&, bool)> _doTests, std::string const& _testString);
 int checkRandomTest(std::function<void(json_spirit::mValue&, bool)> _doTests, json_spirit::mValue& _value, bool _debug = false);
 
 namespace dev { namespace test {
-int createRandomTest(std::vector<char*> const& _parameters)
+int createRandomTest()
 {
-	std::string testSuite;
-	std::string testFillString;
-	json_spirit::mValue testmValue;
-	bool checktest = false;
-	bool filldebug = false;
-	bool debug = false;
-	bool filltest = false;
-
 	dev::test::Options& options = const_cast<dev::test::Options&>(dev::test::Options::get());
-
-	testSuite = options.rCurrentTestSuite;
-	if (testSuite != "BlockChainTests" && testSuite != "TransactionTests" && testSuite != "StateTests"
-				&& testSuite != "VMTests")
-		testSuite = "";
-
-	for (size_t i = 0; i < _parameters.size(); ++i)
-	{
-		auto arg = std::string{_parameters.at(i)};
-
-		if (arg == "--fulloutput")
-			options.fulloutput = true;
-		else if (arg == "--debug")
-			debug = true;
-		else
-		if (arg == "--filldebug")
-			filldebug = true;
-	}
-
-	if (testSuite == "")
+	if (options.rCurrentTestSuite != test::c_StateTestsGeneral)
 	{
-		std::cerr << "Error! Test suite not supported! (Usage -t TestSuite)" << std::endl;
+		std::cerr << "Error! Test suite '" + options.rCurrentTestSuite + "' not supported! (Usage -t <TestSuite>)" << std::endl;
```

Comment:
No. The `\n` is just better. The `std::endl` has a hidden flush included so decreases performance of logs.
https://github.com/isocpp/CppCoreGuidelines/issues/357
http://en.cppreference.com/w/cpp/io/manip/endl